src/storage/src/memtable/btree.rs

```diff
@@ -42,7 +43,7 @@ pub struct BTreeMemtable {
     id: MemtableId,
     schema: RegionSchemaRef,
     map: Arc<RwLockMap>,
-    estimated_bytes: AtomicUsize,
+    stats: MemtableStats,
```

Comment:
out-of-topic: It seems my PR https://github.com/GreptimeTeam/greptimedb/pull/1585 will have some conflicts with yours.

I need to replace the `estimated_bytes` with an `AllocTracker` to track the memory usage of all memtables.